task.sql

```diff
@@ -1,3 +1,15 @@
 USE ShopDB; 
 
 -- Create your stored procedure here
+DELIMITER //
+CREATE PROCEDURE get_warehouse_product_inventory(IN WarehouseID INT)
+BEGIN
+SELECT 
+        p.Name AS ProductName, 
```

Comment:
Why you don't use full name of the field  `Products.Name`. It's make code more readable